packages/python-packages/apiview-copilot/cli.py

```diff
@@ -667,15 +667,23 @@ def get_apiview_comments(review_id: str, environment: str = "production", use_ap
                 raise ValueError("When using the API, `--review-id` must be provided.")
             apiview_endpoints = {
                 "production": "https://apiview.dev",
-                "staging": "https://apiviewstaging.azurewebsites.net",
+                "staging": "https://apiviewstagingtest.com",
             }
             endpoint_root = apiview_endpoints.get(environment)
             endpoint = f"{endpoint_root}/api/Comments/{review_id}?commentType=APIRevision&isDeleted=false"
+            apiview_scopes = {
+                "production": "api://apiview/.default",
+                "staging": "api://apiviewstaging/.default",
```

Comment:
Do we need a UX test scope as well, or does that just use staging scope?

Reply:
this only use staging scope